ext-src/models/IqComponentModel.ts

```diff
@@ -96,12 +96,10 @@ export class IqComponentModel implements ComponentModel {
               this.logger.log(LogLevel.DEBUG, `Getting Internal ID from Public ID: ${this.applicationPublicId}`);
               progress.report({message: "Getting IQ Server Internal Application ID", increment: 40});
               
-              let response: string = await this.requestService.getApplicationId(this.applicationPublicId);
-              this.logger.log(LogLevel.TRACE, `Obtained internal application ID response`, response);
-              
-              let appRep = JSON.parse(response);
+              let internalID: string = await this.requestService.getApplicationId(this.applicationPublicId);
+              this.logger.log(LogLevel.TRACE, `Obtained internal application ID response`, internalID);
         
-              this.requestService.setApplicationId(appRep.applications[0].id);
```

Comment:
This was causing the bad error message @ButterB0wl , etc... (because it was trying to access a property on an empty object, more or less. Moved this into the service, can test for it now, etc...